### ISSUE TITLE
Create state block is too big and take the whole space. 

### ISSUE THREAD

**Issue by obodeldog** [ioBroker/ioBroker.javascript#392]
Please make the "create state" block extendable.

**Comment by AlCalzone** [comment]
Do you have an example what you mean? Screenshot would help.

**Comment by obodeldog** [comment]
Befor:

![image](https://user-images.githubusercontent.com/35403739/63645903-1bb4a480-c708-11e9-8189-c2a0952cf594.png)

Now:
![image](https://user-images.githubusercontent.com/35403739/63645904-28d19380-c708-11e9-8e8e-a4f43071fdf2.png)

**Comment by Apollon77** [comment]
Yes it got enhanced. So what’s the problem beside that it is bigger now and so could screw up „the design“?

**Comment by obodeldog** [comment]
No other Problem :-) Only the desire to improve the clarity again.

**Comment by AlCalzone** [comment]
When seeing this, I think it should be possible to hide the additional inputs behind the additional options button which already exists on some blocks.
